src/app/(auth)/_components/new-password-form.tsx

```
@@ -0,0 +1,184 @@
+                                    </FormControl>
+                                    <FormMessage />
+                                </FormItem>
+                            )}
+                        />
+                    </div>
+                    <Button
+                        type="submit"
+                        // disabled={loading}
+                        className="w-full cursor-pointer"
+                    >
+                        {false ? <Spinner size="sm" /> : "Sign in"}
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)